src/map/script.cpp

```diff
@@ -748,6 +748,21 @@ static unsigned int calc_hash(const char* p)
 	return h % SCRIPT_HASH_SIZE;
 }
 
+bool script_check_RegistryVariableLength(int pType, const char *val, size_t* vlen) 
+{
+	size_t len = strlen(val);
+
+	if (vlen)
+		*vlen = len;
+	switch (pType) {
+		case 0:
+			return (len < 33); // key check
+		case 1:
+			return (len < 255); // value check
```

Comment:
Still think those two sizes should be moved to mmo.h and value should be limited to UINT8_MAX, since that's what actually limits the value, because we send the size of the string as unsigned 8 bit integer.

Reply:
I think not. I mean yes from the prototocol inter-serv point of view he limit is uint8_max, but it won't be limited on that specific case then. All our packet are constructed without check on variable bound.
Now we could add those but it's an overhead and since we receive only our packet atm would that really make sens ?
I mean if at least the packet was created and sent in an async manner that wouldn't impact the server but atm =/.
I feel that since I will change the inter-serv com soon enough it doesn't worth the trouble.

From a system point of view, the limit is really the len of sql.acc_reg_num and sql.acc_reg_str key and value.
Now I could add the attribute len table in a .h, which is probably a bit better, but really from a tiny bit. The current size is localized in same spot atm with that function. A macro won't bring more, as in most time you would want to check against the value of that macro a string (thus the function).
Only somewhat interesting case would be to create array with fixed sized with that macro. (stack array). But if you do this to soon you prevent yourself to detect len error, then within the same process passing an adress or std::move would be better then to have an array so in the end ~

Now current case or macro we still have 2 different place that define the sql table attribute length. Which I feel is not ideal. 
We could at server startup grab that info and set it to a static to check against. (but it's map-serv that would have to ask char-serv the size of his db bound...)  
Or we could have that conf file and make the sql query be based upon this.